lighthouse-cli/test/smokehouse/test-definitions/byte-efficiency/expectations.js

```diff
@@ -169,12 +169,12 @@ const expectations = [
           },
         },
         'uses-responsive-images': {
-          displayValue: 'Potential savings of 69\xa0KB',
+          displayValue: 'Potential savings of 53\xa0KB',
```

Comment:
these values are calculated  based on DPR. They are proportional to `DPR^2`.

`3^2 / 2.625^2 = ~1.3`, so I divided each expectation by 1.3